sections/back_cover.tex

```diff
@@ -5,3 +5,9 @@
     \includegraphics[height=\paperheight, keepaspectratio]{\layout/back.jpg}
   };
 \end{tikzpicture}
+
+\begin{figure*}[!hb]
+  \hypertarget{Difficulty Table}{}
+  \centering
+  \includegraphics[width=\linewidth]{\tables/field_difficulty.png}
```

Comment:
Please crop out the drop shadow of this image. It's no longer necessary on the back cover.